modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/FrontendDataSet.js

```diff
@@ -567,7 +567,7 @@ const FrontendDataSet = ({
 
 	const view =
 		!dataLoading && !componentLoading ? (
-			<div className="data-set-content-wrapper">
+			<div className="data-set-content-wrapper" data-id={id}>
```

Comment:
I'd say we agreed to use a wrapper class here, to keep CSS rules tied to this notation:

```
.wrapper-class-of-the-big-div .cell-my-column {
   min-width: $bar
}
```

isn't it @markocikos ?

Reply:
But the wrapper belongs to the affected commerce pages. Should I add a wrapper for each commerce `frontend-data-set` ?

Reply:
Doesn't minium theme add some top level CSS class?

Reply:
Define theme level column sizes and add a couple of classes to fds wrappers. The problem is that there are several `frontend-data-set-display` and maybe not all of them are affected. Or at least, not the same columns.

Reply:
Honestly, this should no be our consideration. Styling of a product solution is domain of the product team, and they are the ones who should determine what should be effects, and in what scope. They know the cases, UI, and business requirements. You can do the best you can, the rest should be in their hands to tweak.

Reply:
@markocikos Changes related to commerce displays had been reverted. This `data-id` remains as a fds identifier without any reference to styling.

Reply:
Do we need `data-id` for anything?

Reply:
No need. I've removed this change in the followup [PR](https://github.com/liferay-frontend/liferay-portal/pull/4426)

Reply:
Not for this PR, however, I'd like to open a discussion about exposing the FDS id in the markup, something like:
```
<div className={classNames('data-set-content-wrapper', id)}>
```
or
```
<div className="data-set-content-wrapper" id={id}>
```
or perhaps in a higher level <div> that includes the MT, the view and the pagination. This can be used for a variety of purposes, styling amongst others. Do you see any downside in this proposal?